backend/src/test/java/heartbeat/component/E2EUrlGeneratorTest.java

```diff
@@ -0,0 +1,27 @@
+package heartbeat.component;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Profile;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.net.URI;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest
```

Comment:
Do not need @SpringBootTest @Profile("e2e") and ExtendWith(SpringExtension.class)